src/ucp/api/ucp.h

```diff
@@ -1716,6 +1717,14 @@ typedef struct {
                                           Relevant for @a ucp_tag_recv_nbx
                                           function. */
     } recv_info;
+
+    /**
+     * Memory handle for pre-registered buffer.
+     * Used as a hint to speed up memory registration.
```

Comment:
@yosefe I'm not sure why it is "hint".

Reply:
The implementation is not required to use this memory handle for communications, for example:
- bcopy/inline could be used
- some of the transports used to send this message do not support zero copy

Reply:
what about `If the handle is provided, network transport and protocols that rely on network registration can skip the step. As a result the communication request can be executed faster and reduce overall latency of the operation.`

Reply:
How about (bit reworded):
```
If the handle is provided, protocols that require registered memory can skip the registration step. As a result, the communication request overhead can be reduced and it can be completed faster.
```

Reply:
sounds good. @tonycurtis  ?

Reply:
maybe "it" should be "the request" just to be completely clear

Reply:
fixed accordingly